update SPM doc for SQL plan auto capture (#7975)
pingcap/docs: system-variables.md

- This variable is used to set the memory usage threshold of the local cache in the `Apply` operator.
- The local cache in the `Apply` operator is used to speed up the computation of the `Apply` operator. You can set the variable to `0` to disable the `Apply` cache feature.

> ### `tidb_mem_quota_binding_cache` (New in v6.0)

Reviewer: ```suggestion
### tidb_mem_quota_binding_cache (New in v6.0.0)
```

- Scope: GLOBAL
- Default value: `67108864` (64 MiB)
- Range: `[0, 2147483647]`
- Unit: Bytes
- This variable is used to set the threshold of the memory used for caching bindings.

> - If a system creates or captures excessive bindings, resulting in overuse of memory space, TiDB returns a warning in the log. In this case, the cache cannot hold all available bindings or determine which bindings to store. For this reason, some queries may have no bindings. To address this problem, you can increase the value of this parameter, which will increase the memory space used for caching bindings. After modifying this parameter, you need to run `admin reload bindings` to reload bindings and validate the modification.

Reviewer: ```suggestion
- If a system creates or captures excessive bindings, resulting in overuse of memory space, TiDB returns a warning in the log. In this case, the cache cannot hold all available bindings or determine which bindings to store. For this reason, some queries might miss their bindings. To address this problem, you can increase the value of this variable, which increases the memory used for caching bindings. After modifying this parameter, you need to run `admin reload bindings` to reload bindings and validate the modification.
```
